base/applications/mspaint/dib.cpp

```diff
@@ -16,13 +16,26 @@ HBITMAP
 CreateDIBWithProperties(int width, int height)
 {
     BITMAPINFO bmi;
+
     ZeroMemory(&bmi, sizeof(BITMAPINFO));
     bmi.bmiHeader.biSize = sizeof(BITMAPINFOHEADER);
     bmi.bmiHeader.biWidth = width;
     bmi.bmiHeader.biHeight = height;
     bmi.bmiHeader.biPlanes = 1;
-    bmi.bmiHeader.biBitCount = 24;
     bmi.bmiHeader.biCompression = BI_RGB;
+
+    if(paletteWindow.bk){
+        bmi.bmiHeader.biBitCount = 1;
+        bmi.bmiHeader.biClrUsed = 1;
+        bmi.bmiColors[0].rgbGreen=255;
```

Comment:
I'm a bit confused here. Isn't color index 0 normally black and color index 1 white?
Maybe I'm mistaken. I can see that the other index isn't initialized here because of the ZeroMemory above.